charts/trino/templates/configmap-coordinator.yaml

```diff
@@ -102,10 +102,24 @@ data:
   {{- end }}
 
   {{- if contains "PASSWORD" .Values.server.config.authenticationType }}
+  {{- if contains "file" .Values.auth.authMethod }}
   password-authenticator.properties: |
     password-authenticator.name=file
     file.password-file={{ .Values.server.config.path }}/auth/password.db
   {{- end }}
+  {{- if contains "ldap" .Values.auth.authMethod }}
+  password-authenticator.properties: |
+    password-authenticator.name=ldap
+    ldap.allow-insecure={{ .Values.auth.ldap.allowInsecure }}
```

Comment:
These values don't have defaults in `values.yaml`, so add a `| default X` filters here.